themes/gohugoioTheme/layouts/_default/baseof.html

```diff
@@ -9,7 +9,7 @@
 
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
      {{/* NOTE: the Site's title, and if there is a page title, that is set too */}}
-    <title>{{ block "title" . }}{{ .Site.Title }} {{ with .Title }} | {{ . }}{{ end }}{{ end }}</title>
```

Comment:
Also, your original code was inserting *two* spaces in the title.

**Actual**

```
Hugo  | Some Page
```

**Inteded**

```
Hugo | Some Page
```

**Because of the two spaces (s) in the template**

```
                                      s                 s
{{ block "title" . }}{{ .Site.Title }} {{ with .Title }} | {{ . }}{{ end }}{{ end }}
```

Reply:
Firefox (58.0b11) is smart enough to remove the extra space, so you don't actually notice it when browsing, but if you `view source` on [gohugo.io/documentation](http://gohugo.io/documentation/), you'll see 

```html
<title>Hugo  | Hugo Documentation</title>
```